src/gui_common/TweakedColourPicker.tscn

```diff
@@ -1,443 +1,111 @@
+"_editor_description_": "A color picker with thrive theme and extended usage."
+}
+
+[node name="MarginButtonsContainer" type="MarginContainer" parent="."]
+margin_top = 494.0
+margin_right = 299.0
+margin_bottom = 541.0
+custom_constants/margin_top = 5
+
+[node name="ButtonsContainer" type="HBoxContainer" parent="MarginButtonsContainer"]
+margin_top = 5.0
+margin_right = 299.0
+margin_bottom = 47.0
+size_flags_horizontal = 3
+
+[node name="HSVCheckButton" type="CheckButton" parent="MarginButtonsContainer/ButtonsContainer"]
+margin_right = 101.0
+margin_bottom = 42.0
+theme = ExtResource( 1 )
+shortcut_in_tooltip = false
+action_mode = 0
+keep_pressed_outside = true
+text = "HSV"
+__meta__ = {
+"_editor_description_": "PLACEHOLDER"
+}
+
+[node name="RawCheckButton" type="CheckButton" parent="MarginButtonsContainer/ButtonsContainer"]
+margin_left = 105.0
+margin_right = 207.0
+margin_bottom = 42.0
+theme = ExtResource( 1 )
+shortcut_in_tooltip = false
+action_mode = 0
+keep_pressed_outside = true
+text = "Raw"
+__meta__ = {
+"_editor_description_": "PLACEHOLDER"
 }
+
+[node name="HtmlColourStart" type="Label" parent="MarginButtonsContainer/ButtonsContainer"]
+margin_left = 211.0
+margin_top = 10.0
+margin_right = 223.0
+margin_bottom = 32.0
+text = "#"
+__meta__ = {
+"_editor_description_": "PLACEHOLDER"
+}
+
+[node name="HtmlColourEdit" type="LineEdit" parent="MarginButtonsContainer/ButtonsContainer"]
+margin_left = 227.0
+margin_right = 299.0
+margin_bottom = 42.0
+size_flags_horizontal = 3
+theme = ExtResource( 1 )
+align = 1
+max_length = 8
+context_menu_enabled = false
+shortcut_keys_enabled = false
+
+[node name="Separator" type="HSeparator" parent="."]
+margin_top = 541.0
+margin_right = 299.0
+margin_bottom = 549.0
+rect_min_size = Vector2( 0, 8 )
+
+[node name="PresetContainer" type="GridContainer" parent="."]
+margin_top = 549.0
+margin_right = 299.0
+margin_bottom = 549.0
+columns = 12
+
+[node name="PresetButtonContainer" type="MarginContainer" parent="."]
+margin_top = 549.0
+margin_right = 299.0
+margin_bottom = 584.0
+custom_constants/margin_top = 5
+
+[node name="AddPresetButton" type="TextureButton" parent="PresetButtonContainer"]
+margin_top = 5.0
+margin_right = 30.0
+margin_bottom = 35.0
+rect_min_size = Vector2( 30, 30 )
+size_flags_horizontal = 0
+size_flags_vertical = 4
+texture_normal = ExtResource( 3 )
+texture_pressed = ExtResource( 4 )
+texture_hover = ExtResource( 4 )
+texture_disabled = ExtResource( 5 )
+expand = true
+
+[connection signal="color_changed" from="." to="." method="OnColourChanged"]
+[connection signal="toggled" from="MarginButtonsContainer/ButtonsContainer/HSVCheckButton" to="." method="OnHSVButtonToggled"]
+[connection signal="toggled" from="MarginButtonsContainer/ButtonsContainer/RawCheckButton" to="." method="OnRawButtonToggled"]
+[connection signal="focus_exited" from="MarginButtonsContainer/ButtonsContainer/HtmlColourEdit" to="." method="OnHtmlColorEditFocusExited" binds= [ "-" ]]
```

Comment:
There's still binds set to the signal, should be removed as otherwise the signal can't connect to the method.